lib/prepare-params.js

```diff
@@ -53,6 +53,13 @@ module.exports = function(event, context) {
               if (event.request.files) { params[param.name] = event.request.files[param.name]; }
             } else if (event.request.body) {
               params[param.name] = event.request.body[param.name];
+              if (param.type === 'object') {
```

Comment:
these logics are directly stolen, copypastaed, and plagiarized from the query case above